src/appMain/CMakeLists.txt

```diff
@@ -138,6 +150,20 @@ if(ENABLE_LOG)
   list(APPEND LIBRARIES expat -L${EXPAT_LIBS_DIRECTORY})
 endif()
 
+include_directories (
+  ${default_media_inc}
+)
+
+set (SOURCES
+  main.cc
+  life_cycle.cc
+  signal_handlers.cc
+)
+
+if( NOT CMAKE_BUILD_TYPE )
```

Comment:
@AKalinich-Luxoft  this code looks like also redundant. Probably this code is already available in main CMaleLists.txt